packages/transport-axios/src/AxiosTransport.js

```diff
@@ -0,0 +1,33 @@
+import HttpTransport from '@stellarjs/transport-http';
+
+class AxiosTransport extends HttpTransport {
+  constructor(axios, source, log, requestTimeout, baseUrl) {
+    super(source, log);
+    this.defaultRequestTimeout = requestTimeout;
+    this.axios = axios;
+    this.baseUrl = baseUrl;
+  }
+
+  request(req) {
+    return this.send(req);
+  }
+
+  fireAndForget(req) {
+    this.send(req);
```

Comment:
`fireAndForget` means that the message is in the queue, but the caller is not waiting for the response from the handler..
I don't see a point in implementing this function on a "client transport" as it can be achieved with a flag passed as part of the options.

Reply:
I agree but the corrent API for transports includes it.
@sloops77 your thoughts